conf/application.conf

```diff
@@ -19,3 +19,5 @@ credentials = {
   username = "?"
   password = "?"
 }
+
+default-request-queue-name = "PACS001_request"
```

Comment:
I tried that but it doesn't work. I think because we are using [PureConfig which is based on Typesafe Config](https://pureconfig.github.io/) which uses HOCON then this applies: https://github.com/lightbend/config/blob/main/HOCON.md#hyphen-separated-vs-camelcase